api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/PaymentAction.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.dhsc.htbhf.claimant.reporting;
+
+/**
+ * Action that triggered a payment event.
+ */
+public enum PaymentAction {
```

Comment:
Might be good to have a description of what the enum values mean because I have no idea.

Reply:
Added a comment for each value.